examples/game_of_life/Cargo.toml

```
@@ -7,7 +7,7 @@ authors = ["Diego Cardoso <dige0card0s0@hotmail.com>",
 edition = "2018"
 
 [dependencies]
-rand = "0.7.3"
 log = "0.4"
 wasm-logger = "0.2.0"
 yew = { path = "../../yew" }
+getrandom =  { version = "*", features = ["wasm-bindgen"] }
```

As per the [POSIX specification](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206) files should end with new lines.
```suggestion
getrandom =  { version = "*", features = ["wasm-bindgen"] }

```